src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/ListingNotificationHelperTest.java

```diff
@@ -139,6 +139,23 @@ void shouldGetAdjournHearingInstructMessage() {
             hearingInstructMessage.getNote());
     }
 
+    @Test
+    @MockitoSettings(strictness = Strictness.WARN)
+    void shouldGetHearingBundleReadyInstructMessage() {
+
+        Hearing.HearingBuilder hearingData = Hearing.HearingBuilder.hearing()
+            .withHearingType("ORAL")
+            .withHmctsHearingRef("ariaListingReference");
```

Comment:
I don't see this statement being used anywhere??

Reply:
Removed unused statement.